htdocs/product/list.php

```diff
@@ -711,7 +711,8 @@
 		if (! empty($arrayfields['p.ref']['checked']))
 		{
 			print '<td class="tdoverflowmax200">';
-			print $product_static->getNomUrl(1);
+			$pagination = 'page='.$page.'&limit='.$limit.'&contextpage='.$contextpage.'&type='.$type.'&search_type='.$search_type.'&sortfield='.$sortfield.'&sortorder='.$sortorder;
+			print $product_static->getNomUrl(1,'',0,-1,$pagination);
```

Comment:
What is goal of this ? Fix a bug ? Can you describe how to reproduce ?

Reply:
https://github.com/Dolibarr/dolibarr/issues/9625

Reply:
these parameters will be added to the "return list" link to return to the starting point and not to the zero point as is currently the case.